plugins/plugin_compile/project_compile.py

```diff
@@ -475,9 +497,24 @@ def build_android(self):
             ide_name = 'Eclipse'
         cocos.Logging.info(MultiLanguage.get_string('COMPILE_INFO_ANDROID_PROJPATH_FMT', (ide_name, project_android_dir)))
 
+        # Check whether the gradle of the project is support ndk or not
+        gradle_support_ndk = False
+        if using_studio:
+            # Get the engine version of the project
+            engine_version_num = self.get_engine_version_num()
+            if engine_version_num is None:
+                raise cocos.CCPluginError(MultiLanguage.get_string('COMPILE_ERROR_UNKNOWN_ENGINE_VERSION'))
+
+            # Gradle supports NDK build from engine 3.15
+            main_ver = engine_version_num[0]
+            minor_ver = engine_version_num[1]
+            if main_ver > 3 or (main_ver == 3 and minor_ver >= 15):
```

Comment:
It means that the `build.gradle` in Android Studio project supports NDK build if the engine version is larger than 3.15.